.gitignore

```diff
@@ -25,7 +25,42 @@ imgui.ini
 .idea/
 tags
 
-# Do not track QtCreator imported project files
+# Do not track QtCreator imported or any other project files
 OpenBoardView.config
 OpenBoardView.files
 OpenBoardView.includes
+*.creator
+*.creator.*
+.qmake.stash
+*.pro
+*.pro.*
+
+# Dolphin hidden files marker
+.directory
+
+*.a
+
+# https://github.com/github/gitignore/blob/master/CMake.gitignore
+CMakeLists.txt.user
+CMakeCache.txt
+CMakeFiles
+CMakeScripts
+Testing
+Makefile
+cmake_install.cmake
+install_manifest.txt
+compile_commands.json
+CTestTestfile.cmake
+_deps
+
+debug_build
+CPackConfig.cmake
+CPackSourceConfig.cmake
+
+include/version.h
+asset/openboardview.appdata.xml
+asset/openboardview.desktop
+share/appdata/openboardview.appdata.xml
+share/applications/openboardview.desktop
+share/icons/hicolor/scalable/apps/openboardview.svg
+share/mime/packages/openboardview.xml
```

Comment:
Is it worth just ignoring /asset and `/share`?

Reply:
The asset folder contains files in git:
https://github.com/OpenBoardView/OpenBoardView/tree/master/asset

But with the share I agree with you. Give me a minute to fix it.

Reply:
If we already have the directory I'd probably prefer having a .gitignore file _in_ the asset dir

Reply:
Makes sense, fixed it!